http_service/bugbug_http/app.py

```diff
@@ -190,7 +190,29 @@ def schedule_job(
     )
 
 
-def schedule_bug_classification(model_name: str, bug_ids: Sequence[int]) -> None:
+def prepare_queue_job(job: JobInfo, job_id: Optional[str] = None) -> Queue:
+    """Prepare queue data for for enqueueing multiple jobs"""
+    job_id = job_id or get_job_id()
+
+    redis_conn.mset({job.mapping_key: job_id})
```

Comment:
There is some duplication between `schedule_job` and `prepare_queue_job` too.

Reply:
refactored to remove duplication